gruntfile.js

```diff
@@ -216,10 +232,11 @@ module.exports = function(grunt) {
         });
     });
 
-    grunt.registerTask('build', ['eslint', 'clean', 'showcase', 'copy', 'less:development', 'connect:livereload']);
+    grunt.registerTask('build', ['eslint', 'clean', 'showcase', 'copy', 'concat:dist', 'less:development', 'connect:livereload']);
+    grunt.registerTask('build:uglify', ['eslint', 'clean', 'showcase', 'copy', 'uglify', 'less:development', 'connect:livereload']);
     grunt.registerTask('serve', ['build', 'openfin:serve']);
-    grunt.registerTask('createZip', ['build', 'download']);
-    grunt.registerTask('ci', ['build', 'download']);
+    grunt.registerTask('createZip', ['build:uglify', 'download']);
+    grunt.registerTask('ci', ['build:uglify', 'download']);
     grunt.registerTask('deploy', ['ci', 'gh-pages:origin']);
```

Comment:
So `concat` is only run for dev, and `uglify` run for prod, as it concats the files itself?